pkg/columns/filter/filter.go

```diff
@@ -51,6 +54,14 @@ type FilterSpec[T any] struct {
 }
 
 func getValueFromFilterSpec[T any](fs *FilterSpec[T], column *columns.Column[T]) (value reflect.Value, err error) {
+	if column.RawType() == durationType {
```

Comment:
Not a language lawyer, so I don't understand what is meant by
> In particular, since reflect uses reflect.Type equality as type identity, aliases must in general not appear in the reflect runtime data or API

in https://github.com/golang/proposal/blob/master/design/18130-type-alias.md#reflect

@flyth Do you have any ideas?

Reply:
Hmm, as I understand it, this affects definitions like:

```golang
type foo time.Duration
```

A comparison of the reflect.TypeOf() of a var of type `foo` with the reflect.TypeOf() of var of type `time.Duration` would fail afaiu.